mic_vad_streaming/requirements.txt

```diff
@@ -1,4 +1,4 @@
-deepspeech==0.7.0
+deepspeech>=0.7.0
```

Comment:
won't this accept 0.8 ? we only can guarantee 0.7.x

Reply:
Right. This should use [deepspeech~=0.7.0](https://www.python.org/dev/peps/pep-0440/#compatible-release) instead.